<launch>
        <param name="use_sim_time" value="true" />
      	<node pkg="rosbag" type="play" name="player" args="--clock -l /home/federico/multicamera_marker.bag" />        

        <node pkg="image_transport" name="k1_ir" type="republish" args="compressed in:=/k1/ir/image raw out:=/k1/ir/image" />
        <node pkg="image_transport" name="k1_depth" type="republish" args="compressedDepth in:=/k1/depth/image raw out:=/k1/depth/image" />
        <node pkg="marker_tracker" name="k1" type="marker_tracker_node" args="k1" output="screen" />


        <node pkg="image_transport" name="k2_ir" type="republish" args="compressed in:=/k2/ir/image raw out:=/k2/ir/image" />
        <node pkg="image_transport" name="k2_depth" type="republish" args="compressedDepth in:=/k2/depth/image raw out:=/k2/depth/image" />
        <node pkg="marker_tracker" name="k2" type="marker_tracker_node" args="k2" output="screen" />

        <node pkg="image_transport" name="k3_ir" type="republish" args="compressed in:=/k3/ir/image raw out:=/k3/ir/image" />
        <node pkg="image_transport" name="k3_depth" type="republish" args="compressedDepth in:=/k3/depth/image raw out:=/k3/depth/image" />
        <node pkg="marker_tracker" name="k3" type="marker_tracker_node" args="k3" output="screen" />
        
</launch>
